pkgs/development/androidndk-pkgs/androidndk-pkgs.nix

```diff
@@ -79,8 +79,11 @@ rec {
     cc = binaries;
     bintools = binutils;
     libc = targetAndroidndkPkgs.libraries;
+    passthru = { inherit androidndk; };
     extraBuildCommands = ''
-      echo "-D__ANDROID_API__=${stdenv.targetPlatform.sdkVer}" >> $out/nix-support/cc-cflags
+      echo "-D__ANDROID_API__=${stdenv.targetPlatform.sdkVer} " >> $out/nix-support/cc-cflags
+      echo "-target ${stdenv.targetPlatform.config} " >> $out/nix-support/cc-cflags
+      echo "--gcc-toolchain=${androidndk}/libexec/android-sdk/ndk-bundle/toolchains/${targetInfo.triple}-${targetInfo.gccVer}/prebuilt/${hostInfo.double} " >> $out/nix-support/cc-cflags
```

Comment:
This is stuff for LLVM?

Reply:
Yeah androidndk 18+ is using clang + ld.bfd. Apparently they are planning to move lld eventually. Maybe this would be a good chance to get rid of their toolchain completely and just use gccCrossStageStatic?

Reply:
On the nixpkgs side, gcc would be probably easier to support,
since at the moment the defacto standard is: `linux == gcc`.
Does the current cross-compiling port uses bionic or musl on android?

Reply:
It uses bionic which would still make since. Right now we also use the NDKs c compiler which isn’t really necessary although it does save us building gcc for android. Since we have androidndk in hydra now i think it becomes easier to do thay.

Reply:
It uses bionic which would still make since. Right now we also use the NDKs c compiler which isn’t really necessary although it does save us building gcc for android. Since we have androidndk in hydra now i think it becomes easier to do that.